### ISSUE TITLE
Vaginal foreign body sensation

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9868]
**Preferred term label:**

**Synonyms**
Sympathalgia

**Definition (free text, please give PubMed ID)**
PMID 17828787 : Rectal or Vaginal Foreign Body Sensation (Sympathalgia)
Patients generally use fairly vivid terms to describe their deep pain, generally situated in the anus and rectum or
sometimes in the vagina or urethra. They frequently describe a feeling of ‘‘foreign body,’’ but other expressions are also
suggestive: feeling of a stake, a lump, heaviness, a tennis ball, a gnawing, or crawling feeling. This symptom is sometimes
incorrectly called levator ani syndrome in the absence of any correlation with levator ani hypertonia.11 This pain has an
autonomic connotation and temporary relief of these sensations by anesthetic block of sympathetic fibers of the ganglion
impar suggests that they are mediated by sympathetic fibers, justifying the term ‘‘sympathalgia.’’

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pudendal neuralgia ORPHA:60039

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034806
